### PR TITLE
remove s3.region parameter

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/10561, https://github.com/pingcap/docs-cn/pull/10684
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/migrate-from-aurora-bulk-import.md [hunk 1]
```diff
@@ -120,7 +119,6 @@ You need to prepare an EC2 to run the following data export task. It's better to
     --threads 8 \
     -o "$backup_dir" \
     --consistency="none" \
-    --s3.region="$s3_bucket_region" \
```
Comment (ran-huang) [inline tidb-cloud/migrate-from-aurora-bulk-import.md:123]: @WangLe1321 Do we need to remove the s3.region parameter from TiDB cloud docs?

Comment (WangLe1321) [reply tidb-cloud/migrate-from-aurora-bulk-import.md:123]: I think we can retain `s3.region` here. User can install any version of dumpling by tiup. If the version of dumpling is lower than v6.2.0, it also need `s3.region` parameter.

**Comment by WangLe1321** [comment]
Others LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**